docs/website/docs/walkthroughs/add-incremental-configuration.md

```diff
@@ -40,15 +40,14 @@ This strategy technically does not load only new data but instead reloads all da
 Here’s a walkthrough:
 
 1. The initial table, named "contact", in the SQL source looks like this:
-    
+
     | id | name | created_at |
     | --- | --- | --- |
     | 1 | Alice | 2024-07-01 |
     | 2 | Bob | 2024-07-02 |
 
-2. The python code illustrates the process of loading data from an SQL source into BigQuery using the `dlt` pipeline. 
-Please note the `write_disposition = "replace”` used below.
```

Comment:
@dat-a-man  A new line after a numbered list item was the reason why the code block didn't render properly (see the screenshot)

Reply:
Hey @burnash, thanks for looking into it. Do you see some snippets and tables broken in deploy preview here?

Reply:
@dat-a-man no, I think I fixed all of them for this page